package-res/ccc/core/base/panel/panel/panel.js

```diff
@@ -424,6 +424,31 @@ def
         if(this.isRoot) this.chart._onLaidOut();
     },
 
+
+   // CDF603
+   // Getters
+
+    getLayoutSize: function() {
+        return this._layoutInfo ? this._layoutInfo.referenceSize : undefined; //?????
```

Comment:
I don't think so, only clientSize, referenceSize, desiredClientSize and pageClientSize.